ruby/vampires.rb

```diff
@@ -1,5 +1,4 @@
 ## loop
-
 puts "How many employees will be processed at this time?"
 process_times = gets.chomp.to_i
 
```

Comment:
Hey Tyler, like this? 

 puts "How old are you? What year were you born?"
      current_year = Time.now
      employee_age = gets.chomp.to_i
      year_born = gets.chomp.to_i
       age = current_year.year - year_born
      if age == employee_age
         age = true
        else
         age = false
      end